docs/csharp/programming-guide/types/index.md

```diff
@@ -47,7 +47,7 @@ When you declare a variable or constant in a program, you must either specify it
 
 [!code-csharp[csProgGuideTypes#36](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsProgGuideTypes/CS/Class1.cs#36)]
 
-The types of method parameters and return values are specified in the method signature. The following signature shows a method that requires an [int](../../language-reference/builtin-types/integral-numeric-types.md) as an input argument and returns a string:
+The types of method parameters are specified in the method signature. The following signature shows a method that requires an [int](../../language-reference/builtin-types/integral-numeric-types.md) as an input argument:
```

Comment:
I think the same change works better here as well, because the return type is specified:

```suggestion
The types of method parameters and return values are specified in the method declaration. The following signature shows a method that requires an [int](../../language-reference/builtin-types/integral-numeric-types.md) as an input argument and returns a string:
```